src/sections/Navigation/utility/data.js

```diff
@@ -54,10 +54,6 @@ const Data = {
                 {
                     name: "Blog Grid",
                     path: "/blog-grid"
-                },
-                {
-                    name: "Blog List",
-                    path: "/blog-list"
```

Comment:
Don't we want to keep this view as well?

Reply:
Here, we are using that view for the filtered blogs. But, I had a different thought for that view. And will discuss it in the meeting this Thursday. Then, we can decide on that part.

Reply:
ok, @Nikhil-Ladha. Are we good to move forward in the meantime?

Reply:
I have added to the meeting minutes so that we don't forget this discussion.

Reply:
I think we can hold it for a day. And, then we can move forward with what to do.